src/services/plugin/fetch.ts

```diff
@@ -57,3 +58,25 @@ export const fetchPage = async (
   });
   return res;
 };
+
+export const resolveUrl = (
+  pluginId: string,
+  path: string,
+  isNovel?: boolean,
+) => {
+  if (isUrlAbsolute(path)) {
+    return path;
+  }
+  const plugin = getPlugin(pluginId);
+  try {
+    if (!plugin) {
+      throw new Error(`Unknown plugin: ${pluginId}`);
+    }
+    if (plugin.resolveUrl) {
+      return plugin.resolveUrl(path, isNovel);
+    }
+  } catch (e) {
+    throw e;
```

Comment:
```suggestion
    return path;
```
app will crash if plugin is not installed or plugin.resolve doesnt work